### PR TITLE
images removed font-awesome icons added

### PR DESCRIPTION
**Pull request by TanmayBhosale**
font-awesome added and images deleted

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -436,20 +436,30 @@ body {
   width: 40px;
   height: 42px;
   content: '';
-  background-image: url("images/icons/emoji.png");
+  /* background-image: url("images/icons/emoji.png"); */
```
Comment (guptabhaskar) [inline public/style.css]: I think it's better to remove comments.

File: views/index.ejs [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <meta name="description" content="Baat Cheet is an anonymous chat app">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= title %></title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
     <link rel="stylesheet" href="/style.css">
     <link rel="stylesheet" href="/emojify.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (LalitNM) [inline views/index.ejs]: @TanmayBhosale you can remove this.

File: views/index.ejs [hunk 3]
```diff
@@ -28,7 +29,8 @@
         <div class="left">
           <div class="top">
             <input type="text" placeholder="Search..." class="searchtext" />
-            <a onclick="newRoom()" id="newRoom"></a>
+            <!-- added a font-awesome icon -->
```
Comment (LalitNM) [inline views/index.ejs]: This can be changed to `<!-- font-awesome plus --!>`.

**Review by LalitNM — CHANGES_REQUESTED** [review]
In addition to these changes

- you need to delete `package.json` and `package-lock.json` files, as they are just installation files.

- style the send icon.

**Comment by abhigyank** [comment]
@LalitNM the package.json file shouldn't be deleted, but the changes to it in this PR can be removed.

File: views/index.ejs (hunk 2 above)
Comment (TanmayBhosale) [reply views/index.ejs]: if it is removed the icons do not load

Comment (LalitNM) [reply views/index.ejs]: Yeah I just realized that. But there are two `cdn` for `font-awesome`, try to remove the other one.

Comment (TanmayBhosale) [reply views/index.ejs]: done

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
Merged :rocket: Good work @TanmayBhosale.

**Pull request merged**